signer/contentsignaturepki/contentsignature.go

```diff
@@ -110,48 +110,48 @@ func New(conf signer.Configuration) (s *ContentSigner, err error) {
 	if s.db != nil {
 		tx, err = s.db.BeginEndEntityOperations()
 		if err != nil {
-			return nil, errors.Wrap(err, "contentsignaturepki: failed to begin end-entity db operations")
+			return nil, errors.Wrapf(err, "contentsignaturepki %q: failed to begin db operations", s.ID)
 		}
 	}
 	err = s.findAndSetEE(conf, tx)
-	if err != nil {
+	if err == nil {
+		log.Printf("contentsignaturepki %q: reusing existing EE %q", s.ID, s.eeLabel)
+	} else {
+		// No suitable end-entity found, making a new chain
 		if err == database.ErrNoSuitableEEFound {
```

Comment:
I think we could flatten these down to one `else if` ladder or `switch case` though I'm not sure if that's idiomatic

Reply:
funny enough, I did this a few minutes ago because I thought it looked ugly

Reply:
:100: mind reader